packages/ui/src/components/va-skip-link/VaSkipLink.stories.ts

```diff
@@ -0,0 +1,60 @@
+import { VaSkipLink } from './'
+
+export default {
+  title: 'VaSkipLink',
+  component: VaSkipLink,
+}
+
+export const Target = () => ({
+  components: { VaSkipLink },
+  template: `
+    <va-skip-link target="#target">
+      [skip link]
+    </va-skip-link>
+    <div :style="{ height: '100dvh' }">
+      [not target]
+    </div>
+    <div 
+      id="target" 
+      :style="{ height: '100dvh' }"
+    >
+      [target]
+    </div>
+  `,
+})
+
+export const PositionTopLeft = () => ({
```

Comment:
Maybe 4 stories for one prop is too much? We probably can leave only one + have a select or radio to select other positions. (that's an exception, that's not recommended approach in other stories as it's better to just position them side by side, which is not possible for skip link)